### PR TITLE
Jcs activity activity display

### PR DESCRIPTION
**Pull request by JoshSevy**
###### What's this PR do?

Fix and add methods for activity display, in activityRepo.js
Add tests for new methods added
Refactor for failing tests in activityrepo-test.js
Add boilerplate to HTML for the activity/friends card section
Small scss changes for display (add $card-shadow to variables and cards, nested card)

###### Where should the reviewer start?

ActivityRepo.js

###### How should this be manually tested?

npm test

###### Any background context you want to provide?

I still think there are opportunities for more refactor, however the functions are working so can always loop back to those if time permits.

###### What are the relevant tickets?

#53 
#70

### CODE REVIEW

File: test/activityrepo-test.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { expect } from 'chai';
+import { expect, AssertionError } from 'chai';
```
Comment (JoshSevy) [inline test/activityrepo-test.js:1]: I may not need this but added it in when testing for failures in project

File: src/css/_card-design.scss [hunk 2]
```diff
@@ -20,8 +11,20 @@
   display: flex;
   flex-direction: column;
   justify-content: space-between;
+  box-shadow: 6px 5px 8px 6px $card-shadow;
+
+  &-container {
```
Comment (GreysonElkins) [inline src/css/_card-design.scss:16]: oooo fancy - thanks for catching that

File: test/activityrepo-test.js (hunk 1 above)
Comment (GreysonElkins) [reply test/activityrepo-test.js:1]: What's it do?

Comment (JoshSevy) [reply test/activityrepo-test.js:1]: You throw an error on purpose and expect an AssertionError

**Pull request merged**